actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -439,5 +444,6 @@ private void uploadCsvToDB(
       request.setOperation(ActorOperations.PROCESS_BULK_UPLOAD.getValue());
       tellToAnother(request);
     }
+    ProjectLogger.log("BulkUploadManagementActor:uploadCsvToDB method end", LoggerEnum.INFO);
```

Comment:
From:
"BulkUploadManagementActor:uploadCsvToDB method end"

To:
"BulkUploadManagementActor: uploadCsvToDB completed processing for processId "